### PR TITLE
Add ryot

### PR DESCRIPTION
**Pull request by AlexTu2**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/ryot.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+platforms:
+  - Javascript
+  - Docker
+  - Rust
+tags:
+  - Analytics
+  - Miscellaneous
+  - Time Tracking
+depends_3rdparty: false
+demo_url: "ryot.fly.dev"
```
Comment (Ki-er) [inline software/ryot.yml:16]:
```suggestion
demo_url: "https://ryot.fly.dev/auth/login?redirectTo=%2F"
```

File: software/ryot.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - GPL-3.0
+platforms:
+  - Javascript
+  - Docker
+  - Rust
+tags:
+  - Analytics
+  - Miscellaneous
+  - Time Tracking
+depends_3rdparty: false
```
Comment (Ki-er) [inline software/ryot.yml:15]:
```suggestion
```

File: software/ryot.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name: "ryot"
+website_url: "https://github.com/ignisda/ryot"
+source_code_url: "https://github.com/ignisda/ryot"
+description: "Platform for tracking various facets of your life - media, fitness etc."
+licenses:
+  - GPL-3.0
+platforms:
+  - Javascript
+  - Docker
+  - Rust
```
Comment (Ki-er) [inline software/ryot.yml:10]: > https://github.com/ignisda/ryo

```suggestion
platforms:
  - Docker
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Couple suggested changes.

**Comment by AlexTu2** [comment]
> Couple suggested changes.

Changes have been made

File: software/ryot.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+source_code_url: "https://github.com/ignisda/ryot"
+description: "Platform for tracking various facets of your life - media, fitness etc."
+licenses:
+  - GPL-3.0
+platforms:
+  - Docker
+tags:
+  - Analytics
+  - Miscellaneous
+  - Time Tracking
```
Comment (nodiscc) [inline software/ryot.yml:12]:
```suggestion
tags:
  - Personal Dashboards
```


More fitting category

File: software/ryot.yml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+name: "ryot"
+website_url: "https://github.com/ignisda/ryot"
+source_code_url: "https://github.com/ignisda/ryot"
+description: "Platform for tracking various facets of your life - media, fitness etc."
```
Comment (nodiscc) [inline software/ryot.yml:4]:
```suggestion
description: "Platform for tracking various facets of your life - media, fitness, etc."
```

Minor syntax fix

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by AlexTu2** [comment]
@nodiscc changes have been made

**Review by nodiscc — APPROVED** [review]

**Pull request merged**